packages/create-contentful-app/template/src/components/Dialog.spec.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import Dialog from './Dialog';
+import { render } from '@testing-library/react';
+
+describe('Dialog component', () => {
+  it('Matches snapshot', () => {
+    const component = render(<Dialog />);
+
+    expect(component.container).toMatchSnapshot();
```

Comment:
I wonder if we should default to something like

`screen.getByText('Hello Dialog Component')`

instead, because it seems like generally we don't like snapshot testing.